src/main/java/com/softserveinc/ita/models/RandomUtil.java

```diff
@@ -0,0 +1,20 @@
+package com.softserveinc.ita.models;
+
+import com.codeborne.selenide.ElementsCollection;
+import lombok.experimental.UtilityClass;
+import java.util.Random;
+
+import static com.codeborne.selenide.CollectionCondition.*;
+import static java.time.Duration.ofSeconds;
+
+@UtilityClass
+public class RandomUtil {
+
+    public static void randomizerForListCategories(ElementsCollection selector) {
```

Comment:
no, you just need create random method and pass into it size of list:

```
public static int getRandomNumber(int upperBound) {
        return new Random().nextInt(upperBound);
    }
```

or if it lower bound not from 0
```
public static int getRandomNumber(int upperBound) {
        int randomNumber = new Random().nextInt(upperBound);
        randomNumber = randomNumber == 0 ? 1 : randomNumber;

        return randomNumber;
    }
```

Reply:
this is works
public static int getRandomNumber(int upperBound) {
        return new Random().nextInt(upperBound);
    }